debian/build.go

```diff
@@ -38,7 +39,7 @@ func (c cfg) run() error {
 	if err != nil {
 		return err
 	}
-	defer os.RemoveAll(dstdir)
+	//defer os.RemoveAll(dstdir)
```

Comment:
debugging should we allow the package builder to clean these up manually? I'm ok with just deleting this line.